packages/terra-form-select/src/shared/useScrollToActiveOption.js

```diff
@@ -0,0 +1,26 @@
+import { useEffect } from 'react';
+
+export default function useScrollToActiveOption(menuRef, activeDescendantId) {
+  useEffect(() => {
+    const { current: menu } = menuRef;
+
+    if (!menu) {
+      return;
+    }
+
+    const activeOption = document.getElementById(activeDescendantId);
+
+    if (activeOption === null) {
+      return;
+    }
+
+    const dropdownRect = menu.parentNode.getBoundingClientRect();
+    const optionRect = activeOption.getBoundingClientRect();
+
+    if (optionRect.top < dropdownRect.top) {
+      activeOption.scrollIntoView();
```

Comment:
`scrollIntoView` doesn't look to be supported in Safari or IE: https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView#Browser_compatibility

We'll need to either polyfill it or use a different method for scrolling to the active option.

Reply:
This is the same code we used before https://github.com/cerner/terra-core/blob/terra-form-select%405.29.0/packages/terra-form-select/src/search/Menu.jsx#L376-L380

IE and safari don't support `scrollIntoViewOptionsExperimental`, which is the `scrollIntoView({ ... })` API (options hash), but they do support `scrollIntoView()` with/without the boolean `alignToTop` arg

https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView#Parameters